src/wazuh_modules/vulnerability_scanner/tests/component/vulnerabilityScannerFacade_test.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Wazuh Vulnerability Scanner - Unit Tests
+ * Copyright (C) 2015, Wazuh Inc.
+ * September 21, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "vulnerabilityScannerFacade_test.hpp"
+#include "vulnerabilityScannerFacade.hpp"
+#include <external/nlohmann/json.hpp>
+#include <memory>
+
+/*
+ * @brief Log function used by VulnerabilityScannerFacade.
+ */
+// LCOV_EXCL_START
+void logFunction(const modules_log_level_t logLevel, const std::string& logMessage)
+{
+    std::ignore = logLevel;
+    std::ignore = logMessage;
+}
+// LCOV_EXCL_STOP
+
+/*
+ * @brief Test singleton of the VulnerabilityScannerFacade class.
+ */
+TEST_F(VulnerabilityScannerFacadeTest, TestSingletonOfTheVulnerabilityScannerFacadeClass)
+{
+    auto& vulnerabilityScannerFacade = VulnerabilityScannerFacade::instance();
```

Comment:
Improve: Use brace initialization
```suggestion
    auto& vulnerabilityScannerFacade {VulnerabilityScannerFacade::instance()};
```

Reply:
There are several cases like this across all test files.

Reply:
done